pkg/analyze/analyze.go

```diff
@@ -42,8 +43,9 @@ type IPStats struct {
 type Analyzer struct {
 	Config AnalyzerConfig
 
-	ipInfo map[netip.Prefix]IPStats
-	mu     sync.Mutex
+	// [server, ip prefix] -> IPStats
+	serverIpInfo map[serverPrefix]IPStats
```

Comment:
Will we rename this field again when we change the map key in the future? It'd probably be a better idea to use generic names like `stats` and `StatKey`.

With `StatKey` I'd also note: Any reason we keep this entire struct as a private item within the `analyzer` package?

Reply:
No specific reasons for hiding items in struct here, but currently they are not being used outside of `analyze` package.

Also note that `mu` mutex controls `stats`, so I guess it's a better idea to control all reads & writes to stat inside that package, and `StatKey` is used only with `stats`.

Reply:
Exporting `StatKey` will be useful in case we go for something like `a.Lookup(k StatKey)`. As long as we don't export the `stats` field itself, data and mutexes are safe from outside the package.